test/controllers/import_controller_test.rb

```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```

Comment:
If you deleted this folder, you should delete this one, try to not always  use 'rails generate controller'